packages/core/src/__tests__/dispatcher-real-ipfs.test.ts

```diff
@@ -72,7 +73,7 @@ describe('Dispatcher with real ipfs over http', () => {
     const ipfsGetSpy = jest.spyOn(ipfsClient.dag, 'get')
 
     // try to load a CID that ipfs doesn't know about.  It will timeout
```

Comment:
```suggestion
    // try to load a CID that ipfs doesn't know about.  It will timeout.
    // Timeout error message is different depending on if we are talking to a go-ipfs or js-ipfs instance
```